src/app/components/drawer/controlled-drawer.tsx

```diff
@@ -17,6 +17,9 @@ export function ControlledDrawer(props: ControlledDrawerProps) {
 
   return (
     <BaseDrawer
+      height="100vh"
+      position={['absolute', 'fixed']}
+      top={['-24px', 0]}
```

Comment:
Is there a screenshot of what you are fixing here? I'm a bit lost on why I'm not seeing anything broken on my branch with the `BaseDrawer`. 🤔 Maybe I'm just not understanding the change here?

Reply:
I put some screenshots and more explanation in the ticket linked with this commit: 
https://github.com/leather-wallet/extension/issues/4165#issuecomment-1757781322

I'm doing this to close the gap between the top of the modal and the top of the extension: 

https://github.com/leather-wallet/extension/assets/2938440/b0fe260d-d347-4494-83ef-5a38b2a08df4

If you see  that comment, it seems like most of the modals we use have this gap in extension mode. 

https://github.com/leather-wallet/extension/assets/2938440/f9143fcc-1e6f-4516-8444-85bd61d32179

I have this task to add containers - https://github.com/leather-wallet/extension/issues/4165 but on review of that I'm not sure if that will solve this. 

We have issues in extension mode with how we display our modals. If that containers task is to fix the issue we will be changing most modals into full page views which I'm not sure we want to do. 

CC @kyranjamie - maybe we need to catch up on this?